lib/features/reps_core/reps_core.views.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @file
+ * Code for the REPS core feature.
+ */
+ 
+ 
+/**
+ * Implements hook_views_query_alter().
+ */
+ function reps_core_views_query_alter(&$view, &$query) {
+  
+ if ($view->name == 'reps_bean_blocks_global' && $view->current_display=='block_reps_bean') {
+     
+     foreach ($query->where as $index => &$condition_group) {
```

Comment:
Referencing your variable is not needed here: &$condition_group

Reply:
it's strange but yes it's needed... otherwise it is not working. We serach with Michel but no way to escape.